### PR TITLE
show issue/PR numbers together with title

### LINKED ISSUE #1109: Also show issue/PR numbers together with title
## Motivation

It would be nice if issue & PR numbers would somehow also be visible together with the title, perhaps in a darker font so it stands out less than the title (should be in a small column left of the title imho).

Personally, I feel that I frequently filter by PR number mentally (i.e. long-running issues/PRs stand out because I start recognizing the number after a while).

In addition, in EasyBuild we frequently rely on PR numbers to test contributions, via the `eb --from-pr` support we have (cfr. https://easybuild.readthedocs.io/en/latest/Integration_with_GitHub.html#using-easyconfigs-from-pull-requests-from-pr).

## Pitch

Also showing issue/PR number would make it easier to "map" between the octobox overview & the GitHub interface, imho.

### PR DESCRIPTION
**Pull request by chaserx**
Resolves #1109 

**updated**: normal font-weight for the number specifically. 

screen shot 2018-11-11 at 10 15 04 am

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #   actively develop Octobox, you can override this file by adding the following to
 #   a docker-compose.override.yml:
 #
-#   version '3'
+#   version: '3'
```
Comment (chaserx) [inline docker-compose.yml]: i guess this could be in a separate PR, but this tripped me up when setting up

File: docs/INSTALLATION.md [hunk 2]
```diff
@@ -167,7 +167,7 @@ that, you can override the `docker-compose.yml` configuration by adding a
 `docker-compose.override.yml` with the following:
 
 ```yaml
-version '3'
+version: '3'
```
Comment (chaserx) [inline docs/INSTALLATION.md]: same as above

**Comment by boegel** [comment]
@chaserx Thanks a lot for looking into this! Is there any way to make the title stand out a bit more, for example by using a different (dimmer) color for the issue/PR number?

**Comment by chaserx** [comment]
@boegel yeah, that's possible. i'll see what I can do.

**Comment by chaserx** [comment]
@boegel I've updated the screengrab in the description above to show normal font-weight of just the number. thoughts?

**Comment by andrew** [comment]
@chaserx what happens if the repo name has a number in it? Like https://github.com/24pullrequests/24pullrequests

**Comment by chaserx** [comment]
@andrew well... it'll match `24` twice. That's a problem.

**Comment by chaserx** [comment]
@andrew we can get more specific by adding the end of line symbol with something like `\d+$`. 

Example:
http://rubular.com/r/7pmogcVF0i

**Comment by chaserx** [comment]
@andrew it probably wouldn't hurt if i write a couple of tests for this helper method. 😃

**Comment by andrew** [comment]
@chaserx maybe move the number method to the Notification class and unit test it from there?

**Comment by chaserx** [comment]
Moved and tested. Happy to add other test cases.

**Comment by boegel** [comment]
@chaserx Looks better! :+1:

File: docker-compose.yml (hunk 1 above)
Comment (andrew) [reply docker-compose.yml]: Bounced into a new pr: https://github.com/octobox/octobox/pull/1166

File: docs/INSTALLATION.md (hunk 2 above)
Comment (andrew) [reply docs/INSTALLATION.md]: Bounced into a new pr: https://github.com/octobox/octobox/pull/1166

**Comment by chaserx** [comment]
conflicts resolved.

File: app/helpers/notifications_helper.rb [hunk 3]
```diff
@@ -289,6 +289,14 @@ def sidebar_notification_status(status)
     octicon(NOTIFICATION_STATUS_OCTICON[status], height: 16, class: "sidebar-icon #{status}")
   end
 
+  def subject_with_number(notification)
+    if notification.subject_type == 'Issue' || 'PullRequest'
+      "<span class=\"notification-number\">##{notification.subject_number}</span> #{notification.subject_title}".html_safe
```
Comment (andrew) [inline app/helpers/notifications_helper.rb]: Having the HTML concatenated inline feels kinda bad here, especially as it requires using `.html_save` which makes me worry about XSS attacks.

Perhaps instead, in the view we can just conditionally render the number and it's span?

Comment (chaserx) [reply app/helpers/notifications_helper.rb]: ah! yes, I can fix that up this evening.

**Comment by andrew** [comment]
@chaserx brilliant, thanks for updating, shipping it shortly 🎉
